app/server/lib/DocManager.ts

```diff
@@ -638,6 +638,7 @@ export class DocManager extends EventEmitter {
         throw new Error('Grist docs must be uploaded individually');
       }
       const first = uploadInfo.files[0].origName;
+      log.debug(`DocManager._doImportDoc: Received doc with name ${first}`);
```

Comment:
I think my main goal for leaving this in, was to know when a file was traveling along the import route. 

I'll see if I can workspace / org / userId context? Up to you if this is worth having in.

Reply:
The thing is without the context these kinds of logs are mostly useless in prod because they are mixed in with tons of parallel activity. Some kind of context can help you filter, or piece a narrative together. Still, not going to insist on it.